docs/build-parachains.md

```diff
@@ -0,0 +1,174 @@
+---
+id: build-parachains
+title: Parachain Development
+sidebar_label: Parachain Development
+---
+
+This section will cover the motivation to build a parachian or parathread, the tools available
+to facilitate this, and the steps to test, and finally launch your network on Polkadot!
+
+## Why Create a Parachain?
+
```

Comment:
Would be delivered best if the key concepts for creating a parachain were listed first in the section. 

For instance: "pooled security, interoperability, etc.." and perhaps a subheading for each. The page essentially advertises the question of becoming a parachain (i.e. "Why Create a Parachain") - these act as "selling points"